server/app/services/task-service.js

```diff
@@ -98,6 +100,12 @@ async function updateTask(taskId, updatedTask, _embedded) {
 
     if (!currentTask) throw createUserError('UnknownTask', 'This task does not exist');
 
+    if (_embedded && _embedded.kommissionId) {
+        const currentKommission = await Kommission.findById(_embedded.kommissionId);
+
+        if (!currentKommission) throw createUserError('UnknownKommission', 'This kommission does not exist');
+    }
```

Comment:
Should we allow changing the kommission of a task? I don't really see the use case for this

Reply:
you are right, i'm going to change that